rasa/core/processor.py

```diff
@@ -750,7 +749,7 @@ async def parse_message(
     def _parse_message_with_graph(
         self,
         message: UserMessage,
-        tracker: DialogueStateTracker,
+        tracker: Optional[DialogueStateTracker] = None,
```

Comment:
I need help understanding the use of this change here. Is it an optimization?

Reply:
@twerkmeister added it - not sure why he added it but it is a bit cleaner as it is clear when we get a tracker and when there is none. previously we just created an empty tracker in that case.